javascript/reflex_store.js

```diff
@@ -1,3 +1,35 @@
-const reflexes = {}
+const stages = [
+  'created',
+  'before',
+  'delivered',
+  'queued',
+  'after',
+  'finalized',
+  'success',
+  'error',
+  'halted',
+  'forbidden'
+]
+let lastReflex
+
+const reflexes = new Proxy(
+  {}, // You are standing in an open field west of a white house, with a boarded front door.
+  {
+    get: function (target, prop) {
+      if (stages.includes(prop))
+        return Object.fromEntries(
+          Object.entries(target).filter(([_, reflex]) => reflex.stage === prop)
+        )
+      else if (prop === 'last') return lastReflex
+      else if (prop === 'all') return target
+      return Reflect.get(...arguments)
+    },
+    set: function (target, prop, value) {
+      target[prop] = value
+      lastReflex = value
+      return true
+    }
+  }
+)
```

Comment:
Great question, though I propose we table it for a next PR.

There's two high-level approaches possible: 

1. cleanup executed as part of the final stage of each Reflex
2. launch a `setInterval` that runs globally, purges every `n` seconds (configurable via `initialize` option)

Both approaches would execute a callback (with a default provided), with overrides passed as an option to `StimulusReflex.initialize()`.

Which path do you prefer?

As for the default callback, there's lots of ways to slice it:
- do nothing
- nuke reflex(es) right away
- nuke reflex(es) `setTimeout(n)` seconds after they are finalized
- nuke right away if not the most recent Reflex
- nuke by FIFO queue as you suggest, with `n` size configurable

The cool thing is that we really could make the default `() => {}` and then give examples of all of the others in the docs, with an explanation about why they might want to clean up. In some ways, I like defaulting to a null function because of the principle of least surprise.

LMK your thoughts!